test/functionalTest/cases/2015_notification_templates/notification_templates.test

```diff
@@ -337,7 +337,7 @@ User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
 Ngsiv2-Attrsformat: normalized
 Host: 127.0.0.1:REGEX(\d+)
 Accept: application/json
-Content-Type: application/json; charset=utf-8
+Content-Type: text/plain
```

Comment:
Yeah, I also don't know why we do this.
I added charset=utf-8 to text/plain for custom notifications however, in commit 2ac853c.

We can take a look at removing (moving) this strange thing adding charset=utf-8 if json, or including text/plain, but perhaps not in this very PR ?